kuzzle-sdk-java/src/main/java/io/kuzzle/sdk/CoreClasses/Maps/Null.java

```diff
@@ -0,0 +1,8 @@
+package io.kuzzle.sdk.CoreClasses.Maps;
+
+public class Null {
```

Comment:
Nothing except the fact that we need an object that represent a null value without being null, because ConcurrentHashMap can't have null value associated to a key